standard/basic-concepts.md

```diff
@@ -268,7 +268,7 @@ When access to a particular member is allowed, the member is said to be ***acces
 The ***declared accessibility*** of a member can be one of the following:
 
 - Public, which is selected by including a `public` modifier in the member declaration. The intuitive meaning of `public` is “access not limited”.
-- Protected, which is selected by including a `protected` modifier in the member declaration. The intuitive meaning of `protected` is “access limited to the containing class or types derived from the containing class”.
+- Protected, which is selected by including a `protected` modifier in the member declaration. The intuitive meaning of `protected` is “access limited to the containing type or types derived from the containing type”.
```

Comment:
Is there a reason to use type rather than class/interface? (As structs are types and are not derived from)